src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -116,7 +116,7 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         
         List<InstructorAttributes> instructorList = backDoor.getInstructorsForCourse(courseId);
         String joinLink = logic.sendJoinLinkToNewInstructor(instructorList.get(0), data.instructorShortName, data.instructorInstitution);
-        data.statusForAjax = "Instructor " + data.instructorName + " has been successfully created with join link:<br>" + joinLink;
+        data.statusForAjax = "Instructor " + data.instructorName + " created (" + "<a href=" + joinLink + ">join link</a>)";
```

Comment:
Could you check this line again? I think it should be ` "<a href=\"" + joinLink + "\">join link</a>)"` (notice the escaped double quotes for the joinLink